scalafmt-tests/shared/src/test/scala/org/scalafmt/cli/CliOptionsTest.scala

```diff
@@ -104,7 +105,13 @@ class CliOptionsTest extends FunSuite {
     val opt = baseCliOptions.copy(config = Some(configPath.toFile))
     assert(opt.scalafmtConfig.isInstanceOf[Configured.NotOk])
     val confError = opt.scalafmtConfig.asInstanceOf[Configured.NotOk].error
-    assert(confError.cause.exists(_.isInstanceOf[NoSuchFileException]))
+    // native - FileNotFoundException, JVM - NoSuchFileException
```

Comment:
and we can't use the same in both cases?

Reply:
@jchyb If you haven't already, an issue should be filed in Scala Native.